core/src/wallet/bitcoin/BitcoinLikeAccount.cpp

```diff
@@ -438,6 +440,8 @@ namespace ledger {
                 eventPublisher->postSticky(std::make_shared<Event>(code, payload), 0);
                 std::lock_guard<std::mutex> lock(self->_synchronizationLock);
                 self->_currentSyncEventBus = nullptr;
+                span2->close();
```

Comment:
Can getCurrentBlock().onComplete terminate after the span->close()? Is this important?

Reply:
no problem with that :)